src/js/common/uuidGeneratorHelper.js

```diff
@@ -0,0 +1,5 @@
+import uuid from "uuid";
```

Comment:
I'm not sure I like this file name. What else will ever fit into "uuidGeneratorHelper"? I suggest you either make this name much more generic, or, my preference, just use uuid where you actually need it (in mtdFraudPrevention.js)

Abstracting this function out into another file is just going to make reading through the code more difficult as you have to jump around IMO. However if you can come up with a good name for this utility file such that future engineers will probably add to it, I'm okay with that approach too.

Reply:
moved it to mtdFraudPrevention